src/main/java/com/exadel/discountwebapp/vendor/vo/VendorResponseVO.java

```diff
@@ -1,6 +1,6 @@
 package com.exadel.discountwebapp.vendor.vo;
```

Comment:
Let's remove VendorValidator.java empty file for now

Reply:
Removed